.swiftlint.yml

```diff
@@ -0,0 +1,10 @@
+disabled_rules:
+  - trailing_whitespace
```

Comment:
No, this checks if there's spaces at the beginning of a line. We usually have spaces there, and xcode doesn't remove them automatically when the file is saved (as sublime does).